### DISCUSSION TITLE
Empty `Sponsoring` section

### DISCUSSION THREAD

**Discussion by Pavelas** [community/community#18491] [Other Feature Feedback, Questions, & Ideas]
It seems that there's a bug, if you're sponsoring someone privately _(you don't want others to see)_, in your profile empty section `Sponsoring` will appear. I guess, if you're sponsoring someone privately and there are no public sponsors, `Sponsoring` section shouldn't be visible in profile at all.

<details>
<summary>Screenshot 👀</summary>
<img width="327" alt="Screenshot 2022-06-13 at 15 14 34" src="https://user-images.githubusercontent.com/9263565/173351201-1055139e-cad9-4475-8acc-7da0bab8d5c3.png">
</details>

**Comment by sarancodes** [comment]
Try enabling the include private contributions on your profile in contributions and activity settings, if not check the sponsorship log.
